app/assets/stylesheets/widgets.scss

```diff
@@ -0,0 +1,14 @@
+.widget--body {
+  transition: all 0.2s ease;
```

Comment:
Suggestion for consistency (since we use SCSS):

```scss
.widget--body {
  transition: all 0.2s ease;

  &.hiding {
    scale: 0;
    height: 0;
    margin: 0;
    padding: 0;
  }

  &.hidden {
    display: none;
  }
}
```